src/Spryker/Zed/RabbitMq/Business/Model/Queue/QueueInfo.php

```diff
@@ -81,4 +105,15 @@ protected function addRabbitMqQueues(RabbitMqQueueCollectionTransfer $rabbitMqQu
 
         return $rabbitMqQueueCollectionTransfer;
     }
+
+    /**
+     * @param string $currentQueueName
+     * @param array<string> $queueNames
+     *
+     * @return bool
+     */
+    public function isApplicableQueue(string $currentQueueName, array $queueNames): bool
```

Comment:
1. This method is used only in this class, please make it protected and remove it from interface.
2. Since the method name is not very informative,  pleas get rid of it and use in_array instead.